week3/intro.txt

```diff
@@ -1,6 +1,6 @@
 My five favorite things are:
-- 
-- 
-- 
-- 
-- 
+- Family
+- Boyfriend
+- Pets
+- Video Games
```

Comment:
I found it amusing how many (I think everyone) people in this class had video games in common.